core/src/actions/exports.ts

```diff
@@ -108,3 +108,32 @@ export class ExportView extends AuthenticatedAction {
     return { export: await _export.apiData() };
   }
 }
+
+export class ExportsRetryFailed extends AuthenticatedAction {
+  name = "exports:retryFailed";
+  description = "retry failed exports within a timeframe";
+  permission: ActionPermission = { topic: "destination", mode: "write" };
+  inputs = {
+    destinationId: { required: false },
+    startTimestamp: { required: true, formatter: APIData.ensureNumber },
+    endTimestamp: { required: true, formatter: APIData.ensureNumber },
```

Comment:
I was curious if we would have ended up with an `APIData.ensureDate` - but passing timestamps is better! That's what we do on the way out with model.ApiData()